### PR TITLE
Remove dependabot attribution and generate changelog sections using `CL-*` labels

### PR DESCRIPTION
**Pull request by adityapatwardhan**
## PR Summary

* Remove dependabot attribution from change log.
* Create sections in generated change log using the `CL-*` labels.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: tools/releaseTools.psm1 [hunk 1]
```diff
@@ -189,8 +193,20 @@ function Get-ChangeLog
         $new_commits = $new_commits_during_last_release + $new_commits_after_last_release
     }
 
+    $unlabeledPRs = @()
```
Comment (PaulHigin) [inline tools/releaseTools.psm1]: Please add comments explaining each array object.

File: tools/releaseTools.psm1 [hunk 2]
```diff
@@ -211,9 +227,74 @@ function Get-ChangeLog
...
+                "CL-Tools" { $clTools += $commit }
+                "CL-Untagged" { $clUntagged += $commit }
+                "CL-NotInBuild" { continue }
+                Default { throw "unknown tag '$cLabel' for PR: '$($commit.PullRequest)'" }
+            }
+        }
+    }
+
+    if ($multipleLabelsPRs.count -gt 0) {
+        Write-Error "PRs should be tagged with multiple CL labels. PRs with multiple labels: $($multipleLabelsPRs.number -join ' ')"
```
Comment (PaulHigin) [inline tools/releaseTools.psm1]: "... should **not** be tagged..."

File: tools/releaseTools.psm1 [hunk 3]
```diff
@@ -211,9 +227,74 @@ function Get-ChangeLog
...
+        $shouldThrow = $true
+    }
+
+    if ($unlabeledPRs.count -gt 0) {
+        Write-Error "PRs should at least one CL label. PRs missing labels: $($unlabeledPRs.number -join ' ')"
```
Comment (PaulHigin) [inline tools/releaseTools.psm1]: "... should **have** at least one..."

**Review by PaulHigin — CHANGES_REQUESTED** [review]

File: tools/releaseTools.psm1 (hunk 2 above)
Comment (adityapatwardhan) [reply tools/releaseTools.psm1]: Good catch :)

File: tools/releaseTools.psm1 (hunk 3 above)
Comment (adityapatwardhan) [reply tools/releaseTools.psm1]: Fixed.

**Comment by adityapatwardhan** [comment]
@PaulHigin I updated the PRs, please have another look.

**Review by TravisEz13 — APPROVED** [review]

File: tools/releaseTools.psm1 [hunk 4]
```diff
@@ -1,3 +1,5 @@
+#requires -Version 6.0
```
Comment (TravisEz13) [inline tools/releaseTools.psm1]: Our guidance says the copyright has to be first... This is obviously a reasonable exception, but should we update the guidance?

**Review by PaulHigin — APPROVED** [review]
LGTM

**Comment by iSazonov** [comment]
How do we track unlabeled commits and reverted commits?

**Pull request merged**